backup_and_restore/application_backup_and_restore/oadp-intro.adoc

```diff
@@ -0,0 +1,51 @@
+:_content-type: ASSEMBLY
+[id="oadp-introduction"]
+:_content-type: ASSEMBLY
+[id="oadp-introduction"]
+= Introduction to {oadp-full}
+include::_attributes/common-attributes.adoc[]
+:context: oadp-api
+:namespace: openshift-adp
+:local-product: OADP
+
+toc::[]
+= Introduction to {oadp-full}
+:context: oadp-api
+:namespace: openshift-adp
+:local-product: OADP
+
+toc::[]
+
+The {oadp-first} product safeguards customer applications on {product-title}. It offers comprehensive disaster recovery protection, covering OpenShift Container Platform applications, application-related cluster resources, persistent volumes, and internal images. OADP is also capable of backing up both containerized applications and virtual machines (VMs). 
+
+However, OADP does not serve as a disaster recovery solution for xref:../../backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.adoc#backup-etcd[etcd] or OpenShift Operators. 
+
+
+[id="oadp-apis_{context}"]
+= {oadp-full} APIs
+
+{oadp-first} provides APIs to enable multiple approaches to customize backups and prevent the inclusion of unnecessary or inappropriate resources. 
+
+OADP provides the following APIs:
+
+* xref:../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc#backing-up-applications[Backup]
+* xref:../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc#restoring-applications[Restore]
+* Schedule
+* BackupStorageLocation
+* VolumeSnapshotLocation
+
+
+OADP provides APIs to enable multiple approaches to customize a backup to avoid backing up inappropriate resources.
+
+OADP provides the following APIs:
+
+* xref:../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc#backing-up-applications[Backup]
+* xref:../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc#restoring-applications[Restore]
+* Schedule
+* BackupStorageLocation
+* VolumeSnapshotLocation
```

Comment:
Why is [this text repeated](https://github.com/openshift/openshift-docs/pull/64198/files#diff-1ee50b000b47077286254c47ff98fdc2505e4a0b5f2b7c2d4b464d8816860ccfR40-R46)?

Reply:
Cos I made a mistake